integration/shareddata_test.go

```diff
@@ -31,25 +31,16 @@ import (
 )
 
 func TestEnvData(t *testing.T) {
-	notForTigris := []shareddata.Provider{
-		shareddata.Scalars,
-		shareddata.Composites,
-		shareddata.PostgresEdgeCases,
-		shareddata.Nulls,
-		shareddata.ArrayDocuments,
-	}
```

Comment:
> Maybe we could also do something about notForTigris.

@AlekSi here I made change to use compatible provider for tigris, and skip incompatible one instead of having this list.